### DISCUSSION TITLE
Can Adblocker and Links Opener be added for Webview Wallpapers

### DISCUSSION THREAD

**Discussion by imran-prog** [rocksdanister/lively#1173] [Ideas]
Just wanted to know whether lively wallpaper can have Adblocker and Link Opener in the default browser for web Wallpaper.

**Comment by rocksdanister** [comment]
Extensions are not supported by CEF:
https://github.com/ttalvitie/browservice/issues/45

Writing a custom adblocker is an option:
https://www.bilginkilic.com/2020/05/how-to-pre-install-adblock-to-cefsharp.html

In you case you can maybe run system-wide adblocker like pi-hole or use an adblocking dns service like adguard?
